specificatie/geboorte.yaml

```diff
@@ -0,0 +1,59 @@
+openapi: 3.0.3
+info:
+  title: Geboorte definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Geboorte:
+      type: object
+      description: |
+        Gegevens over de geboorte.
+        * **datum** : datum waarop de persoon is geboren.
+        * **land** : land waar de persoon is geboren
+        * **plaats** : gemeente waar de persoon is geboren. Is de persoon geboren buiten Nederland, dan bevat het antwoord alleen een buitenlandse plaatsnaam of aanduiding.
+      properties:
+        datum:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+        land:
+          $ref: 'land.yaml#/components/schemas/AbstractLand'
+        plaats:
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel"
+        inOnderzoek:
+          $ref: "#/components/schemas/AbstractGeboorteInOnderzoek"
+    AbstractGeboorteInOnderzoek:
+      type: object
+      description: |
+        Geeft aan welke gegevens over de geboorte van de persoon in onderzoek zijn. Zie de [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/in_onderzoek.feature)
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaInOnderzoek: '#/components/schemas/GbaGeboorteInOnderzoek'
+          InOnderzoek: '#/components/schemas/GeboorteInOnderzoek'
+    GeboorteInOnderzoek:
+      allOf:
+        - $ref: '#/components/schemas/AbstractGeboorteInOnderzoek'
+        - type: object
+          properties:
+            datum:
+              type: boolean
+            land:
+              type: boolean
+            plaats:
+              type: boolean
+            datumIngangOnderzoek:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    GbaGeboorteInOnderzoek:
+      allOf:
+        - $ref: '#/components/schemas/AbstractGeboorteInOnderzoek'
+        - type: object
+          properties:
+            aanduidingGegevensInOnderzoek:
+              type: string
+              maxLength: 50
```

Comment:
waarom maxLength 50? In LO GBA is dit altijd exact 8 cijfers. Wanneer je dat te light coupled vindt (kan ik me voorstellen), vanwaar dan toch deze maxLength?

Reply:
heb ik uit het Logisch Ontwerp GBA versie 3.12.

v.b. veld: AanduidingGegevensInOnderzoekPersoon, formaat: TXT..50

Reply:
zie https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/pull/896#discussion_r796408578